src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
+    protected ColorSupplier colorSupplier = new ColorSupplier();
+    protected Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(BOUND));
+    }
```

Comment:
great job on constants and working with fields in service classes

if you want to go 1 step further, you can pass `new ColorSupplier()` and `new Random()` objects into the Lottery's constructor from `main()`

Reply:
Thanks! Just commited proposed changes